v6/fetchLists/index.js

```
@@ -0,0 +1,218 @@
+        const networkTokens = {};
+        tokensList.forEach(item => {
+          networkTokens[item.name] = [];
+        });
+        tokens.forEach(token => {
+          token.contract_address = token.contract_address.toLowerCase();
+          token.address = token.contract_address;
+          if (token.icon !== '') {
+            token.icon = IMAGE_PROXY + token.icon;
+            token.icon_png = IMAGE_PROXY + encodeURIComponent(token.icon_png);
+          }
+          delete token.link;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)